app/src/main/java/com/example/traders/watchlist/WatchListState.kt

```diff
@@ -1,3 +1,3 @@
 package com.example.traders.watchlist
 
-data class WatchListState(val someData: String = "Some data")
+data class WatchListState(var cryptoList: List<CryptoInfo>)
```

Comment:
always use `val` as much as possible. `var` may introduce race conditions - it's a case when a variable is changed from two different places in the app.